api/python/tests/integration/test_packages.py

```diff
@@ -0,0 +1,57 @@
+""" Integration tests for T4 Packages. """
+import jsonlines
+import os
+import pytest
+
+from t4 import Package, PhysicalKeyType
+
+from mock import patch
+
+LOCAL_MANIFEST = os.path.join(os.path.dirname(__file__), 'data', 'local_manifest.jsonl')
+REMOTE_MANIFEST = os.path.join(os.path.dirname(__file__), 'data', 't4_manifest.jsonl')
+
+def mock_make_api_call(operation_name):
+    """ Mock boto3's AWS API Calls for testing. """
+    if operation_name == 'GetObject':
+        parsed_response = {'Body': {'foo'}}
+        return parsed_response
+    raise NotImplementedError
+
+def test_read_manifest(tmpdir):
+    """ Verify reading serialized manifest from disk. """
+    pkg = Package.load(open(LOCAL_MANIFEST))
+
+
+    with pytest.raises(NotImplementedError):
+        pkg.get('foo')
+
+    out_path = os.path.join(tmpdir, 'new_manifest.jsonl')
+    pkg.dump(open(out_path, "w"))
```

Comment:
Same here. Here it actually matters cause the data might not get written until the file object is garbage collected at some arbitrary time in the future.